Makefile

```diff
@@ -12,6 +12,8 @@ LOOP_PRIVACY_WEBSITE_URL := $(shell echo $${LOOP_PRIVACY_WEBSITE_URL-"https://ww
 LOOP_LEGAL_WEBSITE_URL := $(shell echo $${LOOP_LEGAL_WEBSITE_URL-"https://www.mozilla.org/about/legal/terms/firefox-hello/"})
 LOOP_PRODUCT_HOMEPAGE_URL := $(shell echo $${LOOP_PRODUCT_HOMEPAGE_URL-"https://www.firefox.com/hello/"})
 FIREFOX_VERSION=48.0
+# Taken from http://hg.mozilla.org/mozilla-central/file/5e14887312d4//old-configure.in#l8032
+MOZ_APP_MAXVERSION=`echo $(FIREFOX_VERSION) | sed "s|\(^[0-9]*\).*|\1|"`.*
```

Comment:
We need to keep it as MOZ_APP_MAXVERSION (due to mozilla-central), but you're right, there's no point in doing the sed. I've changed that, I've also changed install.rdf into its own target, so that we're not unnecessarily firing up the preprocessor when we don't need to.